src/App.js

```diff
@@ -8,7 +8,9 @@ import {
   RightSidebar,
   ThemeToggler,
 } from "Components";
+import MockAPI from "Mockman";
 import { Routes, Route, useLocation } from "react-router-dom";
+import { ToastContainer } from "react-toastify";
 import { HomePage, LandingPage } from "./Pages";
```

Comment:
I would suggest you to use absolute imports much better and easy to understand

Reply:
I am using absolute import this was left out by mistake